app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeFriendUi.kt

```diff
@@ -142,7 +141,10 @@ fun QrCodeScreen(
       }
     }
     BottomNavigationMenu(
-        onTabSelected = { tab -> navigationActions.navigateTo(tab) },
+        onTabSelected = { tab ->
+          navigationActions.navigateTo(tab)
+          viewModel.resetConditions()
```

Comment:
Should this run for every tab on the bottom navigation screen or only when going off of the QR Code screen? If so, then lines 145 and 146 should be switched.

Reply:
thanks, will correct this